users.md

```diff
@@ -1,6 +1,476 @@
 ---
 layout: contribute
-title: Bazel Users
+title: Who is using Bazel
 ---
 
-Find a community curated a list of Bazel users in the [Bazel GitHub wiki](https://github.com/bazelbuild/bazel/wiki/Bazel-Users)
\ No newline at end of file
+*Using Bazel? Send an email to [product@bazel.build](mailto:produc@bazel.build) to get added to the list.
+Feel free to also add your company [on StackShare](https://stackshare.io/bazel)*
+
+This page lists projects and companies that are known to use Bazel. This does not constitute an endorsement.
+
+## Companies using Bazel
+
+### [acqio](https://acqio.com.br)
+
+<!-- img src="https://github.com/bazelbuild/bazel/wiki/acqio_logo.svg" width="150" align="right" -->
```

Comment:
Commented out?

Reply:
Thanks for reminding me.  Acqio was what prompted this PR in the first place. There is no place to host their logo on the wiki.   Added.